test/sql/cte/materialized/dml_materialized_cte.test

```diff
@@ -0,0 +1,26 @@
+# name: test/sql/cte/materialized/dml_materialized_cte.test
+# description: Test DML statements with materialized Common Table Expressions (CTE)
+# group: [materialized]
+
+statement ok
+PRAGMA enable_verification
+
+require noalternativeverify
```

Comment:
Is this necessary here?

Reply:
Currently, yes. Until `MATERIALIZED` CTEs are implemented for `UPDATE/INSERT/DELETE`-statements, alternative verify would lead to non-`MATERIALIZED` CTEs here. Therefore, the tests would be unexpectedly successful.